src/index.html

```diff
@@ -11,7 +11,178 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+    <!-- <h1>HTML Form</h1> -->
+    <form
+      action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+      method="post"
+    >
+      <fieldset class="field">
+        <legend>Personal information</legend>
+        <div class="form__field">
+          <label>
+            Surname:
+            <input
+              type="text"
+              name="surname"
+              autocomplete="off"
+            >
+          </label>
+        </div>
+
+        <div class="form__field">
+          <label>
+            Name:
+            <input
+              type="text"
+              name="name"
+              autocomplete="off"
+            >
+          </label>
+        </div>
+
+        <div class="form__field">
+          <label>
+            How old are You?
+            <input
+              type="number"
+              name="age"
+              min="1"
+              max="100"
+              value="12"
+            >
+          </label>
+        </div>
+
+        <div class="form__field">
+          <label>
+            Full date of birth:
+            <input type="date" name="birth_date">
+          </label>
+        </div>
+
+        <div class="form__field">
+          <label>
+            I accept the agreement
+            <input type="checkbox" name="terms">
+          </label>
+        </div>
+      </fieldset>
+
+      <fieldset class="field">
+        <legend>Registration</legend>
+        <div class="form__field">
+          <label>
+            E-mail:
+            <input
+              type="email"
+              name="email"
+              placeholder="email@example.com"
+              required
+            >
+          </label>
+        </div>
+
+        <div class="form__field">
+          <label>
+            Password:
+            <input
+              type="password"
+              name="password"
+              maxlength="30"
+              minlength="6"
+            >
+          </label>
+        </div>
+      </fieldset>
+
+      <fieldset class="field">
+        <legend>An interesting fact about you!</legend>
+        <div class="form__field">
+          Do you love cats?
+          <label>
+            <input
+              type="radio"
+              name="lovingCats"
+              value="yes"
+            >
+              Yes
+          </label>
+          <label>
+            <input
+              type="radio"
+              name="lovingCats"
+              value="no"
+            >
+              No
+          </label>
```

Comment:
minor one - keep elements of same level on same indent
```suggestion
          <label>
            <input
              type="radio"
              name="lovingCats"
              value="yes"
            >
            Yes
          </label>
          <label>
            <input
              type="radio"
              name="lovingCats"
              value="no"
            >
            No
          </label>
```